worldwind/src/commonMain/kotlin/earth/worldwind/draw/DrawableLines.kt

```diff
@@ -11,7 +11,7 @@ import earth.worldwind.util.kgl.GL_LINES
 import kotlin.jvm.JvmStatic
 
 open class DrawableLines protected constructor(): Drawable {
-    var vertexPoints: FloatBufferObject? = null
+    var vertexPoints: FloatArray = FloatArray(0)
```

Comment:
Ensure you can use both  FloatBufferObject and dynamic buffers in DrawContext - first one is useful for static data

Reply:
Should we add some boolean value to DrawableLine that will be used to determine how to bind the vertex data of the line?

Reply:
Just 2 fields 
```
var vertexPointsBuffer: FloatBufferObject? = null
var vertexPoints: FloatArray = FloatArray(0)
```
Boolean useSharedBuffer = vertexPointsBuffer != null
if (useSharedBuffer) old logic ...
else new logic ...

Reply:
Why do we need support if old approach if it is not effective? Why not just to keep new one?

Reply:
There are 2 usecases - static data and dynamic data. Old approach is fine for static data, new has overheads with static data.

Reply:
DrawableLines was designed to be used i ly as a part of Placwmark for those dynamic vertical lines. So, in this particular drawable, we do not need the support of the old approach. Keep just new buffer.

Reply:
DrawableLines was designed to be used i ly as a part of Placwmark for those dynamic vertical lines. So, in this particular drawable, we do not need the support of the old approach. Keep just new buffer.